### ISSUE TITLE
JSON parse error in response

### ISSUE THREAD

**Issue by sarambasich** [tremendous-rewards/tremendous-ruby#11]
Sometimes, Tremendous sends down what appears to be HTML instead of a JSON response. This happens when we try to create an order. There doesn't seem to be any recognizable pattern other than sometimes it just fails. We receive this error:

```
JSON::ParserError (783: unexpected token at '<!DOCTYPE html>
```
Unfortunately, this in turn breaks our API handling. I've included a `rescue StandardError` block around this `create!` call, but this is not hit and instead the entire process is killed, returning a 500 back to our clients.

This seems similar to #8 but I think perhaps deserves a more fundamental fix is needed.

Please advise, thank you!

**Comment by brianstorti** [comment]
Hey, Stefan. Really sorry for the delay and the issue. We're looking into this.

**Comment by brianstorti** [comment]
@sarambasich A fix for this was just released on version 4.3.2.
